src/main/java/com/sogeti/SoSenseiPsyche/model/Game.java

```diff
@@ -0,0 +1,22 @@
+package com.sogeti.SoSenseiPsyche.model;
+
+
+import java.util.Scanner;
+
+public class Game {
+    public static final int CODE_LENGTH = 4;
```

Comment:
@line 7 the CODE_LENGTH is created here but not used in this file?. You made it static to be used in the Guess class. Would be better to just create it there and also make it private

Reply:
but if we need to use this variable in other classes then I think this is not the best idea.
maybe it's better to leave it in the current class and encapsulate it?